src/components/Scorecards/ScorecardsServicePage/ScorecardsServiceNextRules.tsx

```diff
@@ -77,22 +84,20 @@ export const ScorecardsServiceNextRules = ({
           )}
         </Grid>
         <Grid item>
-          {nextLevel !== undefined ? (
-            <>
-              <Typography variant="subtitle1" className={classes.level}>
-                <b>Next Level: </b> {nextLevel.name}
-                <LoyaltyIcon style={{ color: `${nextLevel.color}` }} />
-                <br />
-                Complete these rules to get to the next level:
-                <br />
-                {nextRules.map(rule => (
-                  <i key={`NextRule-${rule.id}`}>
-                    &#8226; {rule.title ?? rule.expression}
-                    <br />
-                  </i>
-                ))}
-              </Typography>
-            </>
+          {nextSteps.nextLevel !== undefined ? (
+            <Typography variant="subtitle1" className={classes.level}>
+              <b>Next Level: </b> {nextSteps.nextLevel.name}
+              <LoyaltyIcon style={{ color: `${nextSteps.nextLevel.color}` }} />
+              <br />
+              Complete these rules to get to the next level:
+              <br />
+              {nextSteps.rulesToComplete.map(rule => (
+                <i key={`NextRule-${rule.id}`}>
+                  &#8226; {rule.title ?? rule.expression}
```

Comment:
there's a utility for this 😄 `ruleName` just to make sure we don't miss it anywhere